hardware/OpenZWave.cpp

```diff
@@ -1510,6 +1510,10 @@ void COpenZWave::AddValue(const OpenZWave::ValueID& vID, const NodeInfo* pNodeIn
 	std::string vLabel = m_pManager->GetValueLabel(vID);
 	std::string vUnits = m_pManager->GetValueUnits(vID);
 
+	if (vLabel.compare("Unknown") == 0){
```

Comment:
Is it possible to make this compatible with our general coding style?
If (vlabel=="unknown")

Reply:
Done! I plead guilty, I am lousy at C++ - I always expect the syntax to be long and difficult :)